pyproject.toml

```diff
@@ -92,7 +92,7 @@ else:
             '--style',
             'compressed',
             'src/wiki/static/wiki/bootstrap/scss/wiki/wiki-bootstrap.scss',
-            'src/wiki/static/wiki/bootstrap/scss/wiki/wiki-bootstrap.min.css'
+            'src/wiki/static/wiki/bootstrap/css/wiki-bootstrap.min.css'
```

Comment:
This isn't something you had done wrong, @oscarmcm - I was debugging why files ended up in the wrong location, and while doing that I had removed and wrongly restored this line.